src/Util/ProjectionUtil/ProjectionUtil.js

```diff
@@ -0,0 +1,60 @@
+import proj4 from 'proj4';
+import OlProjection from 'ol/proj';
+
+/**
+ * Helper class for ol/proj4 projection handling.
+ *
+ * @class
+ */
+export class ProjectionUtil {
```

Comment:
This util seems to be projectspecific. If we still want it in react-geo `proj4CrsDefinitions` and `proj4CrsMappings` should be removed and instead be passed to the init functions.

Reply:
Now, crs Mappings and definitions are passed to init functions of util, see 059eaa3